src/main/java/cz/xtf/openshift/ResourceFunctions.java

```diff
@@ -23,6 +23,10 @@
 		return pods -> pods.stream().anyMatch(pod -> ResourceParsers.hasPodRestartedAtLeastNTimes(pod, n));
 	}
 
+	public static Function<List<Pod>, Boolean> areNoPodsPresent() {
```

Comment:
This is way to trivial to be here... Also you can achieve the same result by using either `areExactlyNPodsReady(0)` or `areExactlyNPodsRunning(0)`.

Reply:
Same as above

Reply:
Correct. Still this method shouldn't be here since it's way to trivial and only says whether list size is zero. It doesn't even work with resource itself (Could be be `List<X>` instead `List<Pod>`). Also what about `pods.isEmpty()`?